pontoon/base/templates/home.html

```diff
@@ -0,0 +1,175 @@
+{% extends 'base.html' %}
+
+{% block title %}Localize Mozilla{% endblock %}
+
+{% block class %}home{% endblock %}
+
+{% block middle %}
+<section id="main">
+  <div>
+
+    <!-- Pagination -->
+    <ul class="side-navigation">
+      <li class="pagination active" href="#section-1"></li>
+      <li class="pagination" href="#section-2"></li>
+      <li class="pagination" href="#section-3"></li>
+      <li class="pagination" href="#section-4"></li>
+      <li class="pagination" href="#section-5"></li>
+      <li class="pagination" href="#section-6"></li>
+    </ul>
+
+    <!-- Section-1: Introduction -->
+    <section class="snaps section-1 relative" data-section-name="section-1">
+      <div class="introduction neg-margin--large">
+        <div class="title-style">A place to localize Mozilla, fast and easy.</div>
+        <div class="content-style">Join a global community of thousands in making Firefox and various Mozilla products speak to over a hundred of cultures in their native language.</div>
+        <div class="flex flex--wrap">
+          <a class="flex-col-full homepage-button primary-button-cta" href="/teams/">
+            <div>Start Localizing Now</div>
+          </a>
+          <p class="flex-col-full">or</p>
+          <a class="flex-col-full homepage-button secondary-button-cta" href="/tour/">
```

Comment:
Hi @shkamaru  please use `href="{{ url('pontoon.translate.locale.agnostic', 'projects', 'demo', 'all-resources') }}"` as a link to the tour as suggested by matjaz in the PR